src/libraries/Common/src/Interop/Interop.MsQuic.cs

```diff
@@ -0,0 +1,15 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
```

Comment:
I thought you will make it local to the library based on https://github.com/dotnet/runtime/blob/master/docs/coding-guidelines/interop-guidelines.md#file-organization :

The only exception to this should be when an assembly P/Invokes to its own native library that isn't available to or consumed by anyone else, e.g. System.IO.Compression P/Invoking to clrcompression.dll. In such cases, System.IO.Compression should have its own Interop folder which follows a similar scheme as outlined in this proposal, but just for these private P/Invokes.

Reply:
Well, then I don't know if System.IO.Compression was a good example of this then. I don't think it's a local library: https://github.com/dotnet/runtime/blob/4f9ae42d861fcb4be2fcd5d3d55d5f227d30e723/src/libraries/Common/src/Interop/Windows/Interop.Libraries.cs#L39 and https://github.com/dotnet/runtime/blob/4f9ae42d861fcb4be2fcd5d3d55d5f227d30e723/src/libraries/Common/src/Interop/Interop.Brotli.cs

Reply:
Hmmm, compression was changed recently to not follow the example given in our own documentation: https://github.com/dotnet/corefx/pull/36253

Either this change should be rolled back or the documentation updated...

Reply:
I'll let you follow up on that. In the mean time, can you point me to a project that is using local libraries so I can get the right structure.

Reply:
E.g. https://github.com/dotnet/runtime/tree/master/src/libraries/System.IO.Ports/src/Interop/Unix